### ISSUE TITLE
Parser accepts IRIs with invalid scheme

### ISSUE THREAD

**Issue by wouterbeek** [rdfjs/rdfxml-streaming-parser.js#11]
When an IRI contains an invalid scheme (e.g., `%http`), the parser incorrectly accepts it as valid data.  For example, the following document is parsed even though its subject term contains an invalid scheme:

```xml
<?xml version="1.0"?>
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
         xmlns:ex="https://example.com/">
<rdf:Description rdf:about="%https://example.com/s">
  <ex:p>
    <rdf:Description rdf:about="https://example.com/o">
    </rdf:Description>
  </ex:p>
</rdf:Description>
```

**Comment by rubensworks** [comment]
Thanks for reporting, that looks incorrect indeed!

Note to self: follow [RFC 3986 on schemes](https://tools.ietf.org/html/rfc3986#section-3.1):

> scheme      = ALPHA *( ALPHA / DIGIT / "+" / "-" / "." )

**Comment by rubensworks** [comment]
Fixed in v1.2.2.
